docs/content/doc/usage/packages/arch.en-us.md

```diff
@@ -0,0 +1,68 @@
+---
+date: "2016-11-08T16:00:00+02:00"
+title: "Title"
+weight: 10
+toc: true
+draft: false
+menu:
+  sidebar:
+    parent: "packages"
+    name: "Arch"
+    weight: 10
+    identifier: "arch"
```

Comment:
Wrong title and must use new docusaurus structure.

Reply:
@KN4CK3R 
Moved file to new packages folder, fixed title. Are there any other changes to the documentation needed?